arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json

```diff
@@ -238,7 +254,7 @@
           "CognitiveServicesAccounts"
         ],
         "description": "Returns all the resources of a particular type belonging to a resource group",
-        "operationId": "CognitiveServicesAccounts_ListByResourceGroup",
+        "operationId": "Accounts_ListByResourceGroup",
```

Comment:
Actually these Id changes were suggested by AutoRest but why changing operationId is breaking change? Do you mean they will affect generated library?